_includes/manuals/1.19/3.1.3_puppet_versions.md

```diff
@@ -73,7 +66,7 @@ Puppet Server is the application for serving Puppet agents used by default in Pu
   <tr>
     <td>2.0 - 2.1</td>
```

Comment:
@ekohl - would love your input regarding the puppetserver versions we should be supporting.

Reply:
Right now no code has been removed so it'll probably still work.